mintbean-calc/src/App.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
```

Comment:
Really great job cleanly outlining how you're calculating mortgage payments in the App.js. In terms of extensibility, it looks like you're returning essentially the entire form. I wonder what it looks like if you wanted to access some of the data in App in a different component, how that would change the structure of this App.js.

Reply:
you would need to pass that data to the child component via an attribute, so it:
App (parent)
this.state{
data: data
}
<ChildComponent   data={this.state.data} />

and then in the Child component you would access the data by saying

func Child(props){
props.data.whatever
}